client/src/styles/ShareButton.css

```diff
@@ -0,0 +1,10 @@
+.share__button.solid {
+    position: relative;
+    fill: grey;
+    width:20px;
+    
+}
```

Comment:
This can be changed from:
`.share__button.solid`
to:
`.share__icon.solid`
based on the change requested on my other comments.